MvvmCross.Android.Support/Core.UI/MvxAndroidLog.cs

```diff
@@ -0,0 +1,17 @@
+﻿// MvxBindingLog.cs
```

Comment:
Not sure I am a big fan of these duplicated Log classes.

Reply:
We could have one file and link to that in each project, but not sure if that would make it any better. What i want to do is change the log name per project so the logging output makes more sense.

Reply:
Right. Make a new issue which describes this, then we can merge this one so it isn't blocking the rest of the 6.0.0 stuff.